app/static/js/vendor/jquery/jquery.checkboxes-1.2.0.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
```

Comment:
This library is completely necessary. Please remove it. Stick to simple checkboxes.